### ISSUE TITLE
Reverse Engineering tests fail when run from VS with TestDriven.NET

### ISSUE THREAD

**Issue by ajcvickers** [aspnet/EntityFramework#3298]
```
------ Test started: Assembly: EntityFramework.SqlServer.Design.FunctionalTests.dll ------

Test 'Microsoft.Data.Entity.SqlServer.Design.FunctionalTests.ReverseEngineering.SqlServerE2ETests.E2ETest_AllFluentApi' failed:
	System.IO.FileNotFoundException : Could not find file 'C:\aspnet\EntityFramework\test\EntityFramework.SqlServer.Design.FunctionalTests\bin\Debug\ReverseEngineering\ExpectedResults\E2E_AllFluentApi\OneToOneFKToUniqueKeyDependent.expected'.
	at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
	at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
	at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
	at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
	at System.IO.File.InternalReadAllText(String path, Encoding encoding, Boolean checkHost)
	at System.IO.File.ReadAllText(String path)
	ReverseEngineering\FileSystemFileService.cs(42,0): at Microsoft.Data.Entity.Relational.Design.ReverseEngineering.FileSystemFileService.RetrieveFileContents(String directoryName, String fileName)
	ReverseEngineering\FileSet.cs(35,0): at Microsoft.Data.Entity.Relational.Design.FunctionalTests.ReverseEngineering.FileSet.Contents(String filepath)
	ReverseEngineering\FileSet.cs(33,0): at Microsoft.Data.Entity.Relational.Design.FunctionalTests.ReverseEngineering.FileSet.Contents(Int32 index)
	ReverseEngineering\E2ETestBase.cs(52,0): at Microsoft.Data.Entity.Relational.Design.FunctionalTests.ReverseEngineering.E2ETestBase.AssertEqualFileContents(FileSet expected, FileSet actual)
	ReverseEngineering\SqlServerE2ETests.cs(186,0): at Microsoft.Data.Entity.SqlServer.Design.FunctionalTests.ReverseEngineering.SqlServerE2ETests.E2ETest_AllFluentApi()

Test 'Microsoft.Data.Entity.SqlServer.Design.FunctionalTests.ReverseEngineering.SqlServerE2ETests.E2ETest_UseAttributesInsteadOfFluentApi' failed:
	System.IO.FileNotFoundException : Could not find file 'C:\aspnet\EntityFramework\test\EntityFramework.SqlServer.Design.FunctionalTests\bin\Debug\ReverseEngineering\ExpectedResults\E2E_UseAttributesInsteadOfFluentApi\OneToOneFKToUniqueKeyDependent.expected'.
	at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
	at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
	at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
	at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
	at System.IO.File.InternalReadAllText(String path, Encoding encoding, Boolean checkHost)
	at System.IO.File.ReadAllText(String path)
	ReverseEngineering\FileSystemFileService.cs(42,0): at Microsoft.Data.Entity.Relational.Design.ReverseEngineering.FileSystemFileService.RetrieveFileContents(String directoryName, String fileName)
	ReverseEngineering\FileSet.cs(35,0): at Microsoft.Data.Entity.Relational.Design.FunctionalTests.ReverseEngineering.FileSet.Contents(String filepath)
	ReverseEngineering\FileSet.cs(33,0): at Microsoft.Data.Entity.Relational.Design.FunctionalTests.ReverseEngineering.FileSet.Contents(Int32 index)
	ReverseEngineering\E2ETestBase.cs(52,0): at Microsoft.Data.Entity.Relational.Design.FunctionalTests.ReverseEngineering.E2ETestBase.AssertEqualFileContents(FileSet expected, FileSet actual)
	ReverseEngineering\SqlServerE2ETests.cs(140,0): at Microsoft.Data.Entity.SqlServer.Design.FunctionalTests.ReverseEngineering.SqlServerE2ETests.E2ETest_UseAttributesInsteadOfFluentApi()

0 passed, 2 failed, 0 skipped, took 5.82 seconds (xUnit.net 2.1.0 build 3179).
```

**Comment by natemcmaster** [comment]
I'd bet the csproj just doesn't have the right "copy to output" setting for this *.expected file. @lajones ?

**Comment by lajones** [comment]
Fixed - commit f1ce78e.
